migrate.py

```diff
@@ -658,10 +662,10 @@ def format_user(user, options):
     bb_user = "Bitbucket: [{0}](https://bitbucket.org/{0})".format(user['nickname'])
     gh_username = _gh_username(user['nickname'], options.users, options.gh_auth)
     if gh_username is not None:
-        gh_user = "GitHub: [{0}](https://github.com/{0})".format(gh_username)
+        gh_user = ", GitHub: [{0}](https://github.com/{0})".format(gh_username)
```

Comment:
I am not understanding the impact of this... Can you explain / provide example of the bug that this solves?

Reply:
In the [current code](https://github.com/jeffwidman/bitbucket-issue-migration/blob/d83a92b/migrate.py#L664), you'd get this behavior (simplified example):

```python
>>> bb_user = "MyUsername"
>>> gh_username = None
>>> if gh_username is not None:
...   gh_user = "GitHub: [{0}](https://github.com/{0})".format(gh_username)
... else:
...   gh_user = ""
...
>>> print(" (" + bb_user + ", " + gh_user + ")")
 (MyUsername, )
```

Note the trailing comma. This change ensures the comma separator is only added if gh_user will be non-empty.

There are other options, e.g. something like:

```python
usernames=[bb_user]
if gh_username is not None:
  usernames.append("GitHub: [{0}](https://github.com/{0})".format(gh_username))
print(" (" + ', '.join(usernames) + ")")
```

If you prefer that, but this case seemed simple enough to just move the location of the comma.